### DISCUSSION TITLE
Time-Average Temperature in Volume Output?

### DISCUSSION THREAD

**Discussion by ChristianBauerEng** [su2code/SU2#2087] [Q&A]
Hello all,

is it currently possible to output the time-averaged temperatures in the volume outputs?
I'm using the compressible URANS solver and so far only managed to get mean-density and mean-pressure as outputs by specifying the TIME_AVERAGE field in VOLUME_OUTPUT.

Thank you!

Kind regards,

Christian

**Comment by pcarruscag** [comment]
It appears it is not implemented.
If you look for "TIME_AVERAGE" occurrences in the code you'll see what you need to change to get the average temperature.

**Comment by ChristianBauerEng** [comment]
Thank you, will give it a try!
